server/selinux/server/relabel.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/bash
```

Comment:
On fedora, maybe...

On debian:
```
$ which bash
/bin/bash
```

Any reason not to `/usr/bin/env bash` here?